api/models/vpp/interfaces/interface.proto

```diff
@@ -143,10 +143,22 @@ message MemifLink {
 
 // VxlanLink defines configuration for interface type: VXLAN_TUNNEL
 message VxlanLink {
+    message Gpe {
+        enum Protocol {
+            UNKNOWN = 0;
+            IP4 = 1;
+            IP6 = 2;
+            ETHERNET = 3;
+            NSH = 4;
+        };
+        uint32 decap_vrf_id = 4;
+        Protocol protocol = 5;
+    }
     string src_address = 1;  /* source VTEP address */
     string dst_address = 2;  /* destination VTEP address */
     uint32 vni = 3;          /* VXLAN Network Identifier */
     string multicast = 4;    /* multicast interface */
+    Gpe gpe = 5;
```

Comment:
The `VxlanLink` will probably be better readable for users if the `Gpe` message is moved next to `gpe` field and separate from common fields with blank link.

Also, since you already modify the model, could you update the comments here so they use line comments and are defined above respective fields. The reason for this is that currently the comments like this are not generated with Go code, thus missing from Go docs as well.

See the vpp.Interface model that was already updated: https://godoc.org/github.com/ligato/vpp-agent/api/models/vpp/interfaces#Interface